packages/pds/src/pipethrough.ts

```diff
@@ -152,15 +152,24 @@ export const doProxy = async (url: URL, reqInit: RequestInit) => {
     )
   }
   const encoding = res.headers.get('content-type') ?? 'application/json'
-  const repoRevHeader = res.headers.get('atproto-repo-rev')
-  const contentLanguage = res.headers.get('content-language')
-  const resHeaders = noUndefinedVals({
-    ['atproto-repo-rev']: repoRevHeader ?? undefined,
-    ['content-language']: contentLanguage ?? undefined,
-  })
+  const resHeaders = makeResHeaders(res)
   return { encoding, buffer, headers: resHeaders }
 }
 
+const RES_HEADERS_TO_FORWARD = [
+  'atproto-repo-rev',
+  'content-language',
+  'atproto-content-labelers',
+]
+
+const makeResHeaders = (res: Response): Record<string, string> => {
+  const headers = RES_HEADERS_TO_FORWARD.reduce((acc, cur) => {
+    acc[cur] = res.headers.get(cur) ?? undefined
```

Comment:
Is it possible for `headers.get(cur)` to be `null`?  That is the only case I can imagine the `?? undefined` having any effect.  If we want/need to deal with empty strings we could make this `|| undefined`.

Reply:
yeah the return type for `headers.get()` is `string | null` 👍